PowerUp/app/src/main/java/powerup/systers/com/StartActivity.java

```diff
@@ -8,17 +8,17 @@
 import android.view.View;
 import android.widget.ImageButton;
 
+
 public class StartActivity extends Activity {
 
+	boolean previouslyStarted;
```

Comment:
Call booleans starting by `is` or similar so it is easy to see they are boolean from their name. Here `hasPreviouslyStarted`